castai/resource_node_template_test.go

```diff
@@ -236,6 +236,56 @@ Tainted = false
 	)
 }
 
+func Test_flattenNodeAffinity(t *testing.T) {
```

Comment:
can you add a test which would test flattenNodeAffinity returning error?

Reply:
Done.